clippy_dev/src/main.rs

```diff
@@ -51,6 +56,26 @@ fn main() {
                         .help("Checks that util/dev update_lints has been run. Used on CI."),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("new_lint")
+                .about("Create new lint and run util/dev update_lints")
+                .arg(
```

Comment:
Also what about an argument for lint category? We can have that argument default to `nursery` and have it be not required.

Reply:
makes sense to me